### DISCUSSION TITLE
cant use  disableWebPagePreview property

### DISCUSSION THREAD

**Discussion by exrtey** [tdlight-team/tdlight-java#196] [Q&A] [answered]
Updated to the latest version no longer available
disableWebPagePreview
![image](https://github.com/tdlight-team/tdlight-java/assets/125629395/66b0e9cf-7cec-40d4-8261-c14b7a6b3170)

**Comment by cavallium** [comment, marked as answer]
Telegram changed this flag, read the documentation about linkpreviewoptions
